src/Shaders/ShadersInclude/pbrBlockSubSurface.fx

```diff
@@ -270,6 +270,11 @@ struct subSurfaceOutParams
             refractionTransmittance *= cocaLambert(volumeAlbedo, vThicknessParam.y);
         #endif
 
+        #ifdef SS_ALBEDOFORREFRACTIONTINT

```

Comment:
You still have this part running:

// Compute tint from min distance only.
            vec3 volumeAlbedo = computeColorAtDistanceInMedia(vTintColor.rgb, vTintColor.w);
            refractionTransmittance *= cocaLambert(volumeAlbedo, vThicknessParam.y);
        
So volume albedo is physically computed but then override just for one part so I guess you want a #elif from the previous cases ???

Reply:
Also environmentRefraction is multiplied by refractionTransmittance a bit later which would then be equivalent to multiply twice by the albedo or its derived color ?

Reply:
Being outside the `#elif` is intensional. This extension affects the surface only so it's possible to still have a different interior colour. e.g. volumeAlbedo could be green and the surfaceAlbedo could be red but they should both be applied to transmitted light.

Reply:
I modified `environmentRefraction` instead of `refractionTransmittance` only because that's what the `SS_LINKREFRACTIONTOTRANSPARENCY` block did.

Reply:
Ok cool so last question is should it be before or after:
// Decrease Albedo Contribution
        outParams.surfaceAlbedo = surfaceAlbedo * (1. - refractionIntensity);


Sorry for all the questions :-)

Reply:
I think it should be before, like the volume absorption logic. It would essentially look the same as a constant thickness volume absorption.